.env.example

```
@@ -410,7 +410,13 @@ INTERNAL_LLM_API_KEY=""   # Groq key from console.groq.com (free tier); omit for
 # free-lane.ts: both must be set. ~1M tok/day free tier typical.
 CEREBRAS_API_KEY=""
 CONTENT_FREE_LANE_ENABLED="false"
-# Secondary free OpenAI-compat host (Gemma/Nemotron free APIs) after Cerebras fails
+# OpenRouter free — FIRST $0 attempt (best free quality), before Cerebras.
+# The key is the same one Hermes uses (openrouter.ai). Optional model list:
+#   OPENROUTER_MODELS='["google/gemma-4-31b-it:free"]'   (default: gemma-4-31b,
+#   z-ai/glm-5.2, llama-3.1-405b — all :free slugs)
+OPENROUTER_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)